packages/stylelint-plugin/src/config.ts

```diff
@@ -0,0 +1,45 @@
+import type { Config } from 'stylelint';
+
+const config: Config = {
+  plugins: ['@css-modules-kit/stylelint-plugin'],
+  languageOptions: {
+    syntax: {
+      atRules: {
+        value: {
+          // Example:
+          // - `@value a: #123;`
+          // - `@value empty:;`
+          // - `@value withoutSemicolon #123;`
+          // - `@value a from './test.module.css';`
+          // - `@value a, b from './test.module.css';`
+          // - `@value a as aliased_a from './test.module.css';`
+          //
+          // CSS Modules Kit does not support the following for implementation simplicity:
+          // - `@value (a, b) from '...';`
+          // - `@value a from moduleName;`
+          //
+          // ref: https://github.com/css-modules/postcss-icss-values/blob/acdf34a62cc2537a9507b1e9fd34db486e5cb0f8/test/test.js
+          prelude:
+            '<custom-ident> :? <declaration-value>? | [ [ <custom-ident> [ as <custom-ident> ]? ]# from <string> ]',
```

Comment:
In the `@value name: val` syntax, `val` accepts almost any token except semicolons. So I use `<declaration-value>`. This may be strictly wrong, but should cover most use cases.

- https://drafts.csswg.org/css-syntax/#any-value
- https://github.com/postcss/postcss/blob/ff48c29ebb16339188761827701e832f516bfb7d/lib/parser.js#L36
- https://github.com/css-modules/postcss-modules-values/blob/3851eecae2a57b462104dad9e2b47503eea78d21/src/index.js#L6